packages/widgets-internal/package.json

```diff
@@ -43,6 +43,7 @@
     "next": "14.2.3",
     "polished": "^4.2.2",
     "react": "^18.2.0",
-    "styled-components": "6.0.7"
+    "styled-components": "6.0.7",
+    "@types/d3": "^7.4.0"
```

Comment:
did we use `d3` in internal widget?

Reply:
Here is the error
https://vercel.com/pancakeswap/gamification/33YiNFyZUpxzFY1K2RZUN3dHqmt1